Cargo.toml

```
@@ -143,6 +144,7 @@ rust-ini = { version = "0.21.3", optional = true }
 ron = { version = "0.8.1", optional = true }
 json5_rs = { version = "0.4.1", optional = true, package = "json5" }
 corn = { version = "0.10.0", optional = true, package = "libcorn" }
+dotenvy ={ version = "0.15.7", optional = true }
```

```suggestion
dotenvy = { version = "0.15.7", optional = true }
```
(look forward to when `cargo fmt` can reformat `Cargo.toml`)